code/parachain/runtime/dali/src/lib.rs

```diff
@@ -926,6 +926,15 @@ impl pallet_staking_rewards::Config for Runtime {
 pub struct BaseCallFilter;
 impl Contains<Call> for BaseCallFilter {
 	fn contains(call: &Call) -> bool {
+		// Claiming calls are not permitted until Pablo is released.
+		// Remove this filter once this happens.
+		if let Call::Vesting(vesting_call) = call {
```

Comment:
Let's skip this for now as we don't have a conclusion on this, even if we did have a conclusion we can still do this through extrinsics.